CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ When updating the changelog, remember to be very clear about what behavior has c
 and what APIs have changed, if applicable.
 
 ## [Unreleased]
+- Fix bug that if a schema is an enum without any symbols, doc gen should handle it instead of throwing exception.
```

Comment:
What are the use cases of enum withouth symbols?

Reply:
There are quite a few use cases use empty enum as the resource key. For example, resources only support finder methods.

Reply:
(1) I still didn't understand. Can't those enums be replaced by string? What benefit would that bring and why people would choose to use the enum?

(2) I do believe here is a bug to fix but I am not sure if you adding the "EmptyEnum" would be the best solution.

Reply:
1. I think it could be string, but using Enum, like enum None{} could be explicitly telling people no resource key is needed for this resource. That's my impression of using empty enum as the key, you may talk to those customers to understand their motivation more.
2. I was thinking about just return "". This is rest.li doc which is just used as as an example. For example, if the type is int or string we just return some random value. We could explicitly call "emptyEnum" to tell people this is an empty enum.